app/views/verify_email.scala.html

```diff
@@ -25,7 +25,7 @@
     @*TODO: Point to edit email address*@
     <p>
         @messages("verifyEmail.accordion.lineOne")
-        <a href='@controllers.routes.HelloWorldController.helloWorld().url'
+        <a href='@controllers.routes.ConfirmEmailController.show().url'
```

Comment:
Can you point this at the Verify email controller please? Just so when the person who implements it doesn't get confused and thinks this is right